packages/cadl-ranch-specs/http/models/property-types/main.cadl

```diff
@@ -146,3 +146,22 @@ model CollectionsModelProperty is ModelTemplate<InnerModel[]>;
 @route("/collections/model")
 interface CollectionsModel
   extends ModelOperations<CollectionsModelProperty, "[{'property': 'hello'}, {'property': 'world'}]"> {}
+
+// Test a model with a string dictionary property
```

Comment:
I don't think this should be added here. Point of this spec is to make sure all the different types works as the type of a property.(check an `int` work, check a `dictionary` work, etc.) It shouldn't go in details. For that we should have a dedicated spec testing dictionaries

Reply:
Hi @timotheeguerin , cadl `Record<>` type is not covered in existing `property-types/main.cadl`, so I'm trying to add a basic test for it. 

Just want to check `dictionary` can work.
Is it valuable to keep at least one `Record<>` test in this `property-types`?

Reply:
Yeah I think is is valuable to add a single test here to make sure properties and dictionaries work together.

Reply:
and then in `dictionaries/main.cadl` we can have the more in depth test for dictionaries whatever they are